### PR TITLE
Junior Java Dev vacancy

### PR DESCRIPTION
**Pull request by MrJean**

### CODE REVIEW

File: _jobs/junior-java-developer.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    </section>
+
+
+    <section id="three" class="wrapper spotlight style3">
+        <div class="inner">
+            <div class="content">
+                <h2 class="major">Technology stack</h2>
+                <ul>
+                    <li>Spring stack, including Spring Boot, Spring MVC, Spring Data and Spring Cloud</li>
+                    <li>Automated testing (unit testing, integration testing and acceptance testing) of applications doesn't have any secrets for you</li>
```
Comment (yannickdeturck) [inline _jobs/junior-java-developer.html]: I would also mention something about knowing how to work with a mocking framework.

File: _jobs/junior-java-developer.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    <section id="one" class="wrapper spotlight style1">
+        <div class="inner">
+            <div class="content">
+                <h2 class="major">Who are we looking for?</h2>
+                <p>
+                    We strive to remain thought leaders in the ever changing technology market.
+                    JWorks makes this a personal goal of each of our colleagues to stay on top of the game. 
+                    This can be done by following or contributing to numerous <span class="accentuate">WORKSHOPS</span>, coding sessions and webinars we organise, 
+                    but what’s more fun than going to <span class="accentuate">INTERNATIONAL CONFERENCES</span>
+                    such as Spring I/O in <span class="accentuate">BARCELONA</span> or MongoDB World in <span class="accentuate">NEW YORK</span>?
```
Comment (yannickdeturck) [inline _jobs/junior-java-developer.html]: I think it's written as Spring **IO**?

File: _jobs/junior-java-developer.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                    Prefer a <span class="accentuate">MACBOOK</span> with <span class="accentuate">INTELLIJ</span> over a Windows with Eclipse? 
+                    No problem. Want to make an impact on the architectural choices or tools we use? Jump right in!
+
+                </p>
+
+                <ul>
+                    <li>Challenging assignments and projects for <span class="accentuate">Belgian top companies</span></li>
+                    <li>A dynamic <span class="accentuate">high tech</span> environment with focus on knowledge and innovation</li>
+                    <li>An open no-nonsense company culture with room for personal initiative</li>
+                    <li>A great number of <span class="accentuate">workshops</span>, trainings and events in cooperation with Ordina Academy </li>
```
Comment (yannickdeturck) [inline _jobs/junior-java-developer.html]: There's a whitespace right before end tag. 🤓

File: _jobs/junior-java-developer.html (hunk 2 above)
Comment (MrJean) [inline _jobs/junior-java-developer.html]: Nope. http://2017.springio.net/

File: _jobs/junior-java-developer.html (hunk 1 above)
Comment (MrJean) [inline _jobs/junior-java-developer.html]: Please check with Anja

**Pull request merged**